test/TestExpectations.json

```diff
@@ -2133,7 +2133,7 @@
     "testIdPattern": "[launcher.spec] Launcher specs Puppeteer Puppeteer.connect should be able to reconnect to a disconnected browser",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["cdp", "firefox"],
-    "expectations": ["PASS"]
+    "expectations": ["SKIP"]
```

Comment:
this expectation is about CDP and I would not expect changes here

Reply:
reverted expectations